tools/disasm/functions.txt

```diff
@@ -2009,58 +2009,58 @@
     0x8010A430:("Map_Init",),
     0x8010A54C:("Minimap_Draw",),
     0x8010A580:("Map_Update",),
-    0x8010A760:("func_8010A760",),
-    0x8010A7CC:("func_8010A7CC",),
-    0x8010A814:("func_8010A814",),
-    0x8010A85C:("func_8010A85C",),
-    0x8010A8A4:("func_8010A8A4",),
-    0x8010A8EC:("func_8010A8EC",),
-    0x8010A990:("func_8010A990",),
-    0x8010AA54:("func_8010AA54",),
-    0x8010AB30:("func_8010AB30",),
-    0x8010AB94:("func_8010AB94",),
-    0x8010AC00:("func_8010AC00",),
-    0x8010AD24:("func_8010AD24",),
-    0x8010ADD4:("func_8010ADD4",),
-    0x8010AE48:("func_8010AE48",),
-    0x8010AECC:("func_8010AECC",),
-    0x8010AF20:("func_8010AF20",),
-    0x8010AF6C:("func_8010AF6C",),
-    0x8010AF94:("func_8010AF94",),
-    0x8010AFE0:("func_8010AFE0",),
-    0x8010B010:("func_8010B010",),
-    0x8010B06C:("func_8010B06C",),
-    0x8010B0B4:("func_8010B0B4",),
-    0x8010B0DC:("func_8010B0DC",),
-    0x8010B108:("func_8010B108",),
-    0x8010B140:("func_8010B140",),
-    0x8010B180:("func_8010B180",),
-    0x8010B1BC:("func_8010B1BC",),
-    0x8010B22C:("func_8010B22C",),
-    0x8010B284:("func_8010B284",),
-    0x8010B2C0:("func_8010B2C0",),
-    0x8010B300:("func_8010B300",),
-    0x8010B434:("func_8010B434",),
-    0x8010B4A4:("func_8010B4A4",),
-    0x8010B520:("func_8010B520",),
-    0x8010B664:("func_8010B664",),
-    0x8010B7A8:("func_8010B7A8",),
-    0x8010B828:("func_8010B828",),
-    0x8010B878:("func_8010B878",),
-    0x8010B8E4:("func_8010B8E4",),
-    0x8010BB0C:("func_8010BB0C",),
-    0x8010BB6C:("func_8010BB6C",),
-    0x8010BBCC:("func_8010BBCC",),
-    0x8010BC28:("func_8010BC28",),
-    0x8010BC7C:("func_8010BC7C",),
-    0x8010BD48:("func_8010BD48",),
-    0x8010BD90:("func_8010BD90",),
-    0x8010BDDC:("func_8010BDDC",),
-    0x8010BE78:("func_8010BE78",),
-    0x8010BEBC:("func_8010BEBC",),
-    0x8010BEF0:("func_8010BEF0",),
-    0x8010BF24:("func_8010BF24",),
-    0x8010BF58:("func_8010BF58",),
+    0x8010A760:("MsgEvent_Cmd00",),
```

Comment:
Would be nice to keep those updated. Specially since those are being touched anyways